vite.config.js

```diff
@@ -20,8 +20,12 @@ import vue from '@vitejs/plugin-vue'
 import vuetify from 'vite-plugin-vuetify'
 import eslint from 'vite-plugin-eslint'
 import IstanbulPlugin from 'vite-plugin-istanbul'
+import dns from 'dns'
 const path = require('path')
 
+// Workaround https://github.com/cypress-io/cypress/issues/25397
+dns.setDefaultResultOrder('ipv4first')
+
```

Comment:
Would prefer to leave it like this, not restricting it to an env var because that's fiddly (and setting it in bashrc does not affect VSCode tasks which is how I usually run the tests 😞)